docs/adr/0022-grant-output-checkers-view-released-file.md

```diff
@@ -0,0 +1,33 @@
+# 22. Grant Output Checkers Permission to View Released Files
```

Comment:
nit: From looking at other ADRs in this repository, the capitalisation style used in other ADRs is sentence case.

So maybe:

```suggestion
# 22. Grant Output Checkers permission to view released files
```

(That depends on whether it's "Output Checkers" or "output checkers"; see my other comment.)